go.mod

```diff
@@ -282,7 +282,7 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 
 	// Our wasmd branch is: https://github.com/desmos-labs/wasmd v0.30.x-desmos
-	github.com/CosmWasm/wasmd => github.com/desmos-labs/wasmd v0.30.0-desmos.1
+	github.com/CosmWasm/wasmd => github.com/desmos-labs/wasmd v0.30.0-desmos.2
```

Comment:
Update to v0.30.0-desmos.2 to have custom AnyResolver to have the custom format of `Any{}` for desmos-bindings, see:
https://github.com/desmos-labs/wasmd/commit/8fa03330f6d5def43464166f60cb34e7de351cbd


And it also fixes the protoResponse pointer marshaling without reset issue, see:
https://github.com/desmos-labs/wasmd/commit/b5cfa16e978097b2e11835c175ec1818d61a52f8